tests/migration_test.go

```diff
@@ -469,6 +469,84 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 		return uid
 	}
 
+	runMigrationAndCollectMigrationMetrics := func(vmi *v1.VirtualMachineInstance, migration *v1.VirtualMachineInstanceMigration, timeout int) string {
+		var pod *k8sv1.Pod
+		var metricsIPs []string
+		var family k8sv1.IPFamily = k8sv1.IPv4Protocol
+
+		vmiNodeOrig := vmi.Status.NodeName
+		By("Finding the prometheus endpoint")
+		pod, err = kubecli.NewVirtHandlerClient(virtClient).Namespace(flags.KubeVirtInstallNamespace).ForNode(vmiNodeOrig).Pod()
+		Expect(err).ToNot(HaveOccurred(), "Should find the virt-handler pod")
+		for _, ip := range pod.Status.PodIPs {
+			metricsIPs = append(metricsIPs, ip.IP)
+		}
+
+		By("Starting a Migration")
+		Eventually(func() error {
+			migration, err = virtClient.VirtualMachineInstanceMigration(migration.Namespace).Create(migration, &metav1.CreateOptions{})
+			return err
+		}, timeout, 1*time.Second).ShouldNot(HaveOccurred())
+		By("Waiting until the Migration Completes")
+
+		if family == k8sv1.IPv6Protocol {
+			libnet.SkipWhenNotDualStackCluster(virtClient)
+		}
```

Comment:
I think this better be at the beginning of the test

Reply:
right